packages/store/src/secure-store/index.ts

```diff
@@ -4,14 +4,59 @@ import { isAppStudio, errorString } from '../utils';
 import { DummyStore } from './dummy-store';
 import { KeytarStore } from './keytar-store';
 import type { SecureStore } from './types';
+import { join, dirname } from 'path';
+import { homedir } from 'os';
+import { default as fs } from 'fs';
+declare function __non_webpack_require__(m: string): any;
```

Comment:
Is this correct?

Reply:
Thanks for the review.
There was a bug without the webpack coding snd it's needed.

Reply:
Can you please add a comment explaining why this is needed for others, thx/

Reply:
Comment added in f5398a5ad0f273602d9e27797454db4537804680